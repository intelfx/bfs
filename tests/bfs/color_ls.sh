cd "$TEST"
"$XTOUCH" -p scratch/foo/bar/baz
ln -s foo/bar/baz scratch/link
ln -s foo/bar/nowhere scratch/broken
ln -s foo/bar/nowhere/nothing scratch/nested
ln -s foo/bar/baz//qux scratch/notdir
ln -s scratch/foo/bar scratch/relative
mkdir scratch/__bfs__
ln -s /__bfs__/nowhere scratch/absolute

export LS_COLORS="or=01;31:"
invoke_bfs scratch/{,link,broken,nested,notdir,relative,absolute} -color -type l -ls \
    | sed 's/.* -> //' \
    | sort >"$OUT"
diff_output
