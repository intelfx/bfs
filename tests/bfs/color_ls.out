[01;31mscratch/foo/bar[0m
[01;31mscratch/foo/bar[0m
[01;34m/[0m[01;31m__bfs__/nowhere[0m
[01;34m/[0m[01;31m__bfs__/nowhere[0m
[01;34mfoo/bar/[0m[01;31mnowhere[0m
[01;34mfoo/bar/[0m[01;31mnowhere[0m
[01;34mfoo/bar/[0m[01;31mnowhere/nothing[0m
[01;34mfoo/bar/[0m[01;31mnowhere/nothing[0m
[01;34mfoo/bar/[0mbaz
[01;34mfoo/bar/[0mbaz
[01;34mfoo/bar/baz[0m[01;31m//qux[0m
[01;34mfoo/bar/baz[0m[01;31m//qux[0m
